### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by kral2** [xchapter7x/clarity#32] [General]
Hello,

Is this project still maintained? Not that I have found something not working as expected. I am starting to review some Terraform testing frameworks before adopting one for my projects, and clarity stands out by its usability.

Just trying to define if it is under active development, maintenance only, or considered feature complete with no plan to evolve anymore, as I can't see lot of commit/issue activities in the last couple of months.

**Comment by xchapter7x** [comment]
hi @kral2 

thanks for reaching out. 
I am open to new feature development, accepting PRs, bugfixes, etc. 

new development will be as active as the community using it :)
